src/github_changelog/git.clj

```diff
@@ -21,16 +21,16 @@
       (git/git-clone uri dir))))
 
 (defn- refresh [^Repository repo]
-  (git/git-fetch-all repo)
-  repo)
-
-(defn clone [config]
-  (let [{:keys [git-url dir update?]
-         :or   {git-url (gen-url config)
-                dir     (git-path git-url)
-                update? true}} config
-        repo                   (clone-or-load git-url dir)]
-    (if update? (refresh repo) repo)))
+  (git/git-fetch-all repo))
+
+(defn clone [{:keys [git-url dir update?]
+              :or   {git-url (gen-url config)
+                     dir     (git-path git-url)
+                     update? true}
+              :as config}]
+  (let [repo (clone-or-load git-url dir)]
+    (if update? (refresh repo))
+    repo))
```

Comment:
This can be rewritten with [`cond->`](http://jakemccrary.com/blog/2016/04/10/the-usefulness-of-clojures-cond-arrow/):
```clojure
(cond-> (clone-or-load git-url dir)
  update? refresh)
```
It expands to:
```clojure
(macroexpand '(cond-> (clone-or-load git-url dir) update? refresh))
; (let* [G__1356 (clone-or-load git-url dir) G__1356 (if update? (clojure.core/-> G__1356 refresh) G__1356)] G__1356)